src/readOutputFilter.m

```diff
@@ -0,0 +1,43 @@
+function outputFiltered = readOutputFilter(cfg, filterHeader, filterContent)
+    % outputFiltered = readOutputFilter(cfg, filterHeader, filterContent)
+    %
+    % It will display in the command window the content of the `output.tsv' filtered by one element
+    % of a target column. At the moment it works only for string content and can retrieve the tsv
+    % path form `cfg`.
+    %
+    % DEPENDENCIES:
+    % - bids_matlab (from CPP_BIDS)
+    %
+    % INPUT:
+    %
+    %  - cfg: the main experiment structure
+    %  - filterHeader: string, the column header where the ctarget content is stored (e.g., for
+    %    'condition name' will be 'trial type')
+    %  - filterContent: string, the content of the column you want to filter out. It can take just
+    %    part of the content name (e.g., you want to display the triggers and you have
+    %    'trigger_motion' and 'trigger_static', 'trigger' as input will do)
+    %
+    % OUTPUT:
+    %
+    %  - outputFiltered: dataset with only the specified content, to see it in the command window
+    %  use display(outputFiltered)
+
+
+    % Get the outputfile path
+    tsvFile = fullfile(cfg.dir.outputSubject, ...
+        cfg.fileName.modality, ...
+        cfg.fileName.events);
+
```

Comment:
After this maybe add a check to make sure the file exists. 

Something along those lines.

```suggestion
if ~exist(tsvFile, 'file')==2
  error('input file does not exist')
end
```